rep-0152.rst

```diff
@@ -0,0 +1,147 @@
+
+In ROS system, we use 'Unmaintained' package status([4]_) and ROS Orphaned Packages Group([5]_), who do not have to take full responsibility to package maintenance jobs, but expecting to take minimum responsibilities such as just to releasing into newer ROS distros.
+
+
+Rationale
+=========
+
+REP-0140 [6]_ and REP-0141 [7]_ provide maintainer information in package manifest file and maintainer status in ROS distribution file.
+
+Maintenance Guide in ROSWiki [8]_ provides how to claim maintainership for orphaned packages.
+
+Operations
+==========
+
+1-A. Orphaning a package by maintainers
+---------------------------------------
+
+If you can no longer maintain a your package.
+
+a. Set the package maintainer section in the ``package.xml`` file as follows.
+
+   ``<maintainer email="ros-orphaned-packages@googlegroups.com">ROS Orphaned Package Maintainers</maintainer>``
+
+b. Send email to ros-orphaned-packages@googlegroups.com and/or post to the `ROS Discourse Release Category <http://discourse.ros.org/c/release>`_, indicating that package is now orphaned.
+   Wait response from someone in the ROS Orphaned Package Maintainer team.
+
+c. Add admin permission of both source and release repository to who respond from in the ROS Orphaned Package Maintainers team.
+
+1-B. Orphaning a package by users
+---------------------------------
+
+If you find a package is not actively maintained and willing to release as a ROS Orphaned Package Maintainer.
+
+a. Contact the current maintainer, ask for transition to orphaned package and request access to source and release repository in order to make releases.
+
+b. If the current maintainer is not responsible (a single day isn't enough to conclude that) and you still want to release the packages, there are two different paths forward depend on the "location" of the source and release repositories.
+
+   1. If the repositories are hosted on GitHub within an `ros-` org unit, the admins of the org unit will usually grant the necessary access to these repositories.
+
+   2. If nobody else beside the original author (which doesn't respond) has access, you should fork the repository.
+
+c. Add ROS Orphaned Package Maintainers to the maintainer section of package.xml as follows.
+
+
+   ``<maintainer email="ros-orphaned-packages@googlegroups.com">ROS Orphaned Package Maintainers</maintainer>``
+
+2. Adopting packages
+--------------------
+
+Orphaned packages are seeking a new maintainer.
+If you willing to step forward the package maintainer, more than just to release into new ROS distros, but respond to the issues, pull requests and so on.
+Contact to the ROS Orphaned Package Maintainers by sending email to ros-orphaned-packages@googlegroups.com and/or post to the `ROS Discourse Release Category <http://discourse.ros.org/c/release>`_.
```

Comment:
Have either a mailing list or (preferably) a Discourse category. We can subscribe to Discourse categories so there is no need for a separate mailing list. If a mailing list is needed to provide an email for the maintainer tag, then either find out if Discourse can provide that sort of feature or have just a mailing list rather than a Discourse category.

Reply:
@gbiggs you're right. Send a message to mailing list is not a good idea over Discourse. Changed to `Post a message to the `ROS Discourse Release Category <http://discourse.ros.org/c/release>`_ indicating that package is now orphaned.`